src/modules/update.js

```diff
@@ -0,0 +1,55 @@
+const taskContainer = document.querySelector('.taskList');
```

Comment:
**[OPTIONAL]**
-_Remember to DRY, in this case, you created the same **taskContainer** calling class **.taskList** in both modules, you can export-import one to don't define twice the variable._